config/logstash.yml

```diff
@@ -286,8 +286,17 @@
 # log.level: info
 # path.logs:
 #
-
-
+# ------------ Password Policy --------------
```

Comment:
We should be opinionated about the default password policy, to simplify users having to configure them as needed, and leave the mode as WARN by default.

For example, our password policy, should be:

Minimum length: 8 characters.
Included Uppercase
Lowercase
Special characters.
And WARN users.